less/core/accessibility.less

```diff
@@ -3,33 +3,28 @@
 .component { position: relative; }
 .block { position: relative; }
 
-// jquery.a11y start
-// Invisible aria-labels
-// --------------------------------------------------
-.aria-label {
+.aria-label, .visually-hidden {
+  display: block;
   position: absolute !important;
-  &.relative {
-    position: relative !important;
-  }
-  left: 0 !important;
-  width: auto !important;
-  height: auto !important;
-  overflow: auto !important;
-  color: rgba(0,0,0,0) !important;
-  background: rgba(0,0,0,0) !important;
-  font-size: 1px !important;
+  width: 1px !important;
+  height: 1px !important;
   padding: 0 !important;
-  margin: 0 !important;
-  line-height: normal !important;
-  z-index: 1;
+  margin: -1px !important;
+  overflow: hidden !important;
+  clip: rect(0, 0, 0, 0) !important;
+  white-space: nowrap !important;
+  border: 0 !important;
+  pointer-events: none;
 }
 
-// Force aria-labels to hide on hidden
-// --------------------------------------------------
-.aria-label.aria-hidden {
-  display: none !important;
+.aria-label-focusable, .visually-hidden-focusable {
+ &:not(:focus):not(:focus-within) {
+    .visually-hidden
```

Comment:
`.visually-hidden` includes `pointer-events: none;` which we probably want to exclude if we intend to use this for clickable buttons e.g. "skip navigation". In which case we could utilise a mixin here. Please see suggestion below.

```
.visually-hidden() {
  display: block;
  position: absolute !important;
  width: 1px !important;
  height: 1px !important;
  padding: 0 !important;
  margin: -1px !important;
  overflow: hidden !important;
  clip: rect(0, 0, 0, 0) !important;
  white-space: nowrap !important;
  border: 0 !important;
}

.aria-label {
  .visually-hidden();
  pointer-events: none;
}

.aria-label-focusable, .visually-hidden-focusable {
 &:not(:focus):not(:focus-within) {
    .visually-hidden;
  }
}
```

Reply:
Done